packages/gatsby/cache-dir/page-renderer.js

```diff
@@ -16,9 +17,17 @@ class PageRenderer extends React.Component {
       loader: publicLoader,
     })
 
+    let PageComponent
+
+    if (process.env.NODE_ENV === `production`) {
+      PageComponent = this.props.pageResources.component
+    } else {
+      PageComponent = loadable(() => this.props.pageResources.component())
+    }
```

Comment:
Not sure if anyone has an opinion on this. page-renderer.js is being used for both development and production but because we're using `react-hot-loader-loader` in development only (in webpack.config.js), loadable needs to be used here in development as well.